src/components/ArticleCard.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react'
+import { makeStyles } from '@material-ui/core/styles'
+import Card from '@material-ui/core/Card'
+import CardHeader from '@material-ui/core/CardHeader'
+import CardActionArea from '@material-ui/core/CardActionArea'
+import CardActions from '@material-ui/core/CardActions'
+import CardContent from '@material-ui/core/CardContent'
+import CardMedia from '@material-ui/core/CardMedia'
+import Grid from '@material-ui/core/Grid'
+import Button from '@material-ui/core/Button'
+import Typography from '@material-ui/core/Typography'
+import Divider from '@material-ui/core/Divider'
+import { styles } from '../theme/styles'
+import Link from '@material-ui/core/Link'
+
+const useStyles = makeStyles(styles)
+
+const ArticleCard = ({ article, spacing }) => {
+  const classes = useStyles()
+
+  return (
+    <Grid item xs={12} sm={6} md={4} spacing={spacing} className={classes.articleCardGrid}>
+      <Card className={classes.articleCardRoot} raised variant='outlined'>
+        <div className={classes.articleContent}>
+          <CardHeader subheader={article.section} className={classes.articleCardHeader} />
+          <CardActionArea>
```

Comment:
The [`CardActionArea`](https://material-ui.com/api/card-action-area/) component renders a `button` element with a pointer cursor and ripple on click. Yet, there's no action associated. I think this might be confusing for users. Consider adding an action or removing the `CardActionArea` i.e. the [Complex Interaction example](https://material-ui.com/components/cards/#complex-interaction).

Reply:
I noticed that as well and I think it has something to do with the :hover styling. One of Material UI's many tricks, I suppose.